### DISCUSSION TITLE
DRM nulled

### DISCUSSION THREAD

**Discussion by cgolib** [heliomarpm/udemy-downloader-gui#52] [Ideas]
si interrumpes la descarga del archivo de video antes de terminar, el video es visible. estaria bien un opcion para detener las descargas antes de xxx bytes hasta ver el punto exacto.

traslate:
if you interrupt the download of the video file before it is finished, the video is visible. An option to stop downloads before xxx bytes until the exact point is seen would be nice

.

**Comment by heliomarpm** [comment]
I've noticed that some files download so quickly that it's not possible to pause.

**Comment by espaol** [comment]
It doesn't work anymore, I have tried a lot to login with Gmail but it doesn't anymore. Anyone faced something like that?

**Reply by heliomarpm** [reply]
Answered
https://github.com/heliomarpm/udemy-downloader-gui/discussions/28#discussioncomment-4186436
